extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/PermissionSetConfig.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.vertx.http.runtime;
+
+import java.util.List;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class PermissionSetConfig {
```

Comment:
Can that be renamed to `RolePermissionSetConfig` or `DefaultPermissonSetConfig`. If say an OIDC adapter would deny an access because the client ip is blacklisted then it won't about the roles ?

Reply:
This implementation is all about roles. The OIDC adaptor will likely not use these classes.